integ/s3/Dockerfile

```diff
@@ -1,12 +1,7 @@
-# Start from the latest golang base image
-FROM public.ecr.aws/amazonlinux/amazonlinux:latest
-RUN curl -sL -o /bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
-RUN chmod +x /bin/gimme
-RUN yum upgrade -y && yum install -y tar gzip git
-ENV HOME /home
-RUN /bin/gimme 1.17.9
-ENV PATH ${PATH}:/home/.gimme/versions/go1.17.9.linux.arm64/bin:/home/.gimme/versions/go1.17.9.linux.amd64/bin
+FROM public.ecr.aws/docker/library/golang:1.17.9 AS base
```

Comment:
Why not latest? This will be forgotten over time to update I would guess

Reply:
I think go has a concept of latest stable which we could use? Is that the best practice? Thus far we've always explicitly locked to a version but then it gets out of date and we get asked to upgrade it...

Reply:
Yes we can use the latest version. I used the hard-coded version since the same was used earlier.